packages/server/src/handlers/BaseHandler.ts

```diff
@@ -33,20 +33,53 @@ export class BaseHandler extends EventEmitter {
   }
 
   generateUrl(req: http.IncomingMessage, id: string) {
-    id = encodeURIComponent(id)
-
-    const forwarded = req.headers.forwarded as string | undefined
-    const path = this.options.path === '/' ? '' : this.options.path
-    // @ts-expect-error baseUrl type doesn't exist?
+    // @ts-expect-error
```

Comment:
To fix the CI warning
```suggestion
    // @ts-expect-error req.baseUrl does exist
```